default.nix

```diff
@@ -1,6 +1,2 @@
-(import (
-  fetchTarball {
-    url = https://github.com/edolstra/flake-compat/archive/f012cc5092f01fc67d5cd6f04999f964c3e05cbf.tar.gz;
-    sha256 = "1n8q7v7alq802kl3b6zan6v27whi2ppbnlv8df6cz64vqf658ija"; }) {
-      src = builtins.fetchGit ./.;
-}).defaultNix
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
Making `default.nix` and `shell.nix` depend on `<nixpkgs>` is a bad idea. 1) It's inefficient, since now evaluation requires at least *2* versions of Nixpkgs. 2) It goes against the whole point of flakes, which is hermetic evaluation. It reintroduces a dependency on `$NIX_PATH` which is what we want to get rid of.